src/main/java/de/uni_tuebingen/ub/nppm/servlet/gast/SharedHtmlServlet.java

```diff
@@ -44,25 +45,30 @@ protected void generatePage(HttpServletRequest request, HttpServletResponse resp
         if (myFile != null && !myFile.endsWith(".html")) {
             myFile += ".html";
         }
-
         try {
             // Inhalt aus der Datenbank basierend auf dem Dateinamen und der Sprache holen
             Content content = ContentDB.getByNameAndLanguage(myFile, selectedLanguage);
-
             if (content != null) {
                 response.setContentType("text/html; charset=UTF-8");
-
                 // Inhalt senden
                 PrintWriter writer = response.getWriter();
                 byte[] htmlBytes = content.getContent();
                 writer.write(new String(htmlBytes, "UTF-8"));
                 writer.flush();
             }
 
-        } catch (Exception e) {
-            // Fehlerbehandlung benutze Standard Sprache Deutsch, da für nicht alle wie Hilfe.html eine Englische Version vorhanden ist
-            Content content = ContentDB.getByNameAndLanguage(myFile, "de");
-
+        } catch (NoResultException e) {
+            Content content = null;
+            try{
+                //Versuche erst englisch zu holen
+                content = ContentDB.getByNameAndLanguage(myFile, "gb");
+            }catch(NoResultException e1){
+                // Fehlerbehandlung benutze Standard Sprache Deutsch, da für nicht alle wie Hilfe.html eine Englische Version vorhanden ist
+                content = ContentDB.getByNameAndLanguage(myFile, "de");
```

Comment:
There should be a constant defined for `de` as well, e.g. `Constants.FALLBACK_LANG`